docs/source/development.rst

```diff
@@ -25,9 +25,17 @@ libraries. On Debian or Ubuntu, we can install these using
 
     $ sudo apt-get install python-dev zlib1g-dev git libxslt1-dev
 
-.. note::
-    TODO: Document this basic step for other platforms? We definitely want
-    to tell people how to do this with Brew or ports on a Mac.
+Alternatively, on Mac computers, install Anaconda Python
```

Comment:
Hm, do we want to tie ourselves to a third party package group like Anaconda?  It's probably better just to stick with a basic package manager like brew...